Packs/MicrosoftGraphTeams/Integrations/MicrosoftGraphTeams/MicrosoftGraphTeams.py

```diff
@@ -0,0 +1,736 @@
+
+    def create_chat(self, subject: str, members: list, user_id: str = None, type='group') -> dict:
+        """Create a new chat for a given user
+
+        Args:
+            subject (str):
+            members (list):
+            user_id (str):
+            type (str):
+
+        Returns:
+            dict:
+        """
+        user_id = user_id if user_id else self.delegated_user
+        suffix = 'chats'
+
+        json_data = {
+            'chatType': type,
+            'topic': subject,
+            'members': self._build_members_input(user_id, members)
+        }
+        return self.ms_client.http_request(method='POST', url_suffix=suffix, json_data=json_data)
+
+    def get_chat(self, chat_id: str, user_id: str = None) -> dict:
+        """Create a new chat for a given user
+
+        Args:
+            chat_id (str):
+            user_id (str):
+
+        Returns:
+            dict:
+        """
+        user_id = user_id if user_id else self.delegated_user
+        suffix = f'users/{user_id}/chats/{chat_id}'
+
+        demisto.debug(f"URL suffix is {suffix}")
+        response = self.ms_client.http_request(method='GET', url_suffix=suffix)
+        return response
+
+    def update_chat(self, chat_id: str, subject: str) -> dict:
+        """Create a new chat for a given user
+
+        Args:
+            chat_id (str):
+            subject (str):
+
+        Returns:
+            dict:
+        """
+        suffix = f'chats/{chat_id}'
+
+        json_data = {
+            'topic': subject
+        }
+        return self.ms_client.http_request(method='PATCH', url_suffix=suffix, json_data=json_data)
+
+    def list_members(self, chat_id: str, user_id: str = None) -> Union[dict, list]:
+        """Returning all members from given chat
+
+        Args:
+            chat_id (str):
+            user_id (str):
+
+        Returns:
+            dict or list:
+        """
+        user_id = user_id if user_id else self.delegated_user
+        suffix = f'users/{user_id}/chats/{chat_id}/members'
+
+        demisto.debug(f"URL suffix is {suffix}")
+        response = self.ms_client.http_request(method='GET', url_suffix=suffix)
+        return response
+
+    def add_member(self, chat_id: str, user_id: str, share_history: bool) -> bool:
+        """Add a member to a given chat
+
+        Args:
+            chat_id (str):
+            user_id (str):
+            share_history (bool):
+
+        Returns:
+            bool:
+        """
+        suffix = f'chats/{chat_id}/members'
+
+        json_data = {
+            '@odata.type': '#microsoft.graph.aadUserConversationMember',
+            'roles': ['owner'],
+            'user@odata.bind': f"https://graph.microsoft.com/v1.0/users('{user_id}')",
+            'visibleHistoryStartDateTime': '0001-01-01T00:00:00Z' if share_history else ''
+        }
+        self.ms_client.http_request(method='POST', url_suffix=suffix, json_data=json_data, resp_type="text")
+
+        return True
```

Comment:
if the chat/member is not valid or the member already an exists it will create an error, and the error message will show the reason for failure. there is no message return except for the status and error message.